src/js/components/MaskedInput/stories/CustomBox.js

```diff
@@ -5,7 +5,7 @@ import { grommet } from 'grommet/themes';
 export const CustomBoxMaskedInput = () => {
   const [value, setValue] = React.useState('');
   const [box, setBox] = React.useState();
-  const boxRef = React.useCallback(setBox, []);
+  const boxRef = React.useCallback(setBox, [setBox]);
```

Comment:
Why isn't this: `const boxRef = React.useRef();`?

Reply:
Updated. Unnecessary in this instance.